extensions/wikia/WikiaRSS/WikiaRssExternalController.class.php

```diff
@@ -127,6 +133,12 @@ private function getFullItemList($items, $options) {
 			if( $item['description'] ) {
 				$text = trim(mb_convert_encoding($item['description'],$outputEncoding,$charset));
 				$text = str_replace( array("\r", "\n", "\t", '<br>'), ' ', $text );
+				$text = strip_tags( $text, '<img><a><br>' );
```

Comment:
Name for `removeHTMLtags()` is very misleading. In fact it's trying to modify HTML in a way that it's safe.